test/gtest/common/test_helpers.h

```diff
@@ -664,6 +664,10 @@ class handle {
     { \
         _t h; \
         ucs_status_t status = _ctor(__VA_ARGS__, &h); \
+        if (status == UCS_ERR_UNSUPPORTED) { \
```

Comment:
we don't want to allow it for *every* type of handle
maybe add new macro like UCS_TEST_CREATE_HANDLE_IF_SUPPORTED